openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -1118,7 +1121,7 @@ def process(self, instance):
             "fps": context.data.get("fps", None),
             "source": source,
             "user": context.data["user"],
-            "version": context.data["version"],  # this is workfile version
```

Comment:
tbh this change might have been required for my other feature where I'm allowing TrayPublisher to submit publishes to the farm https://github.com/ynput/OpenPype/pull/5195 as there it doesn't make sense to have a workfile version. I'm going to revert the change on this PR, although I'd like to understand how is this `version` used later on the publish because that's not what becomes the instance version right?

Reply:
done